backend/build.gradle.kts

```diff
@@ -191,6 +191,7 @@ dependencies {
     }
     implementation("org.springframework.boot:spring-boot-starter-undertow")
     implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
+    implementation("org.springframework.boot:spring-boot-starter-webflux")
```

Comment:
@Googleton , what part of webflux did you use?

Reply:
I used it early on while testing something but forgot to remove it. Will fix!